setup.py

```diff
@@ -55,7 +55,7 @@ def get_version(*file_paths):
         "XBlock>=1.2.2",
         "celery>=3.1",
         "edx-celeryutils>=0.1.5",
-        "edx-completion>=0.1.5",
+        "edx-completion>=1.0.3,<2",
```

Comment:
Here's the key piece.  edx-completion is up-to-date with the latest release, and can flex a bit with updates as needed by edx-platform.